pkg/collector/python/datadog_agent.go

```diff
@@ -609,3 +610,25 @@ func ObfuscateMongoDBString(cmd *C.char, errResult **C.char) *C.char {
 	// memory will be freed by caller
 	return TrackedCString(obfuscatedMongoDBString)
 }
+
+// SendAgentTelemetry records a telemetry data point for a Python integration
+//
+//export SendAgentTelemetry
+func SendAgentTelemetry(checkName *C.char, metricName *C.char, metricValue *C.float) {
```

Comment:
What calls this function? I also think that name "semantically" is a bit wrong. Technically you are not "sending" telemetry, you are simply updating a few metrics. Orthogonally per configuration it will be sent on a certain scheduler

Reply:
I see "minus all the bridge boilerplate) and would appreciate any feedback" that it is coming :).

Reply:
Renamed to `emit`, which I think is better semantically.

The expected call would be something like:
```
datadog_agent.emit_agent_telemetry("postgres", "activity_latency", (time.time() - start_time) * 1000)
```